tests/python/ghost_types/input.py

```diff
@@ -0,0 +1,33 @@
+#! Testing mass() and Z() for both ghost types: A (Gh(x) or @(x)) and B (extract_subsets())
+import psi4
+
+m_He = 4.00260325415 #TEST
+m_Gh = 0.0           #TEST
+Z_He = 2.0           #TEST
+Z_Gh = 0.0           #TEST
+
+frag = psi4.geometry("""
+    He
+    --
+    He 1 1
+""")
+
+A_frag = psi4.geometry(""" 
+    He
+    --
+    Gh(He) 1 1
+""")
+
+B_frag = A_frag.extract_subsets(1,2)
```

Comment:
B_frag is rather just echoing the explicit ghosting set up by A_frag. Did you mean `B_frag = frag.extract_subsets(1, 2)` for testing the other type of ghosting? And maybe throw in a NRE and total mass check for mixed extract_subsets/`@` ghosting with `zero_ghosts` T/F?

Reply:
Yes, absolutely meant `frag.extract_subsets`. I even had that in my test jupyter notebook, not sure how I missed it here, thanks. I'll increase the tests to look at a mixed-ghost case, and total masses. What do you mean by `NRE`?

Reply:
nuclear_repulsion_energy

Reply:
Done, looking much better!